ts/features/bonus/bpd/screens/details/transaction/BpdEmptyTransactionsList.tsx

```diff
@@ -1,14 +1,44 @@
 import { View } from "native-base";
 import * as React from "react";
+import { InfoBox } from "../../../../../../components/box/InfoBox";
 import { Body } from "../../../../../../components/core/typography/Body";
+import { H4 } from "../../../../../../components/core/typography/H4";
+import { IOColors } from "../../../../../../components/core/variables/IOColors";
 import Markdown from "../../../../../../components/ui/Markdown";
 import I18n from "../../../../../../i18n";
 
+const CSS_STYLE = `
+body {
+  font-size: 16;
+  color: ${IOColors.black}
+}
+`;
+
 const BpdEmptyTransactionsList: React.FunctionComponent = () => (
   <>
-    <Body>{I18n.t("bonus.bpd.details.transaction.detail.empty")}</Body>
+    <InfoBox iconSize={32} alignedCentral={true}>
```

Comment:
```suggestion
    <InfoBox  iconName={"io-warning"} iconSize={32} alignedCentral={true}>
```
I guess the icon is the new one

Reply:
@Undermaken Maybe we should change the default icon in InfoBox component then, what do you think?

Reply:
Maybe we should discuss that change with designers team. Since it is used wisely I prefer a local.change and then check it in a dedicated story.
I'm feel more confident 😃